### PR TITLE
[OT276-77] Endpoint Creación Comentarios 

### PR DESCRIPTION
**Pull request by Juanchini23**
### Motivation and Summary
COMO usuario
QUIERO dar de alta nuevos Comentarios
PARA agregarlos a un post existente

Criterios de aceptación:

POST /comments - Deberá validar que se envíe el campo post_id,user_id y body

### Type of change
- [X ] New feature
- [ ] Bug fix
- [ ] New Tests (no functional change)
- [ ] Refactor (no functional change)

### Checklist

- [ ] Traceability between this change and Jira.
- [ ] ```mvn clean install``` was run and all tests completed successfully.
- [ ] There are no unused variables and/or imports in the modified classes.
- [ ] There are no imports in the modified classes with the wildcard character. Ex: ```com.somepackage.*```.
- [ ] Slf4j was used for the application log instead ```System.out```.
- [ ] Public methods are documented with ```javadoc```.

### CODE REVIEW

File: src/main/java/com/alkemy/ong/core/usecase/impl/CommentServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+@RequiredArgsConstructor
+public class CommentServiceImpl implements CommentService {
+
+    private final CommentRepository commentRepository;
+
+    private final NewRepository newRepository;
+
+    @Override
+    @Transactional
+    public Long createEntity(Comment comment, Long idNew) {
```
Comment (manerajona) [inline src/main/java/com/alkemy/ong/core/usecase/impl/CommentServiceImpl.java]: llamar a `idNew` como `newId` para mantener constancia por favor

File: src/main/java/com/alkemy/ong/core/usecase/CommentService.java [hunk 2]
```diff
@@ -4,6 +4,6 @@
 
 public interface CommentService {
 
-	Long createEntity(Comment entity);
-	
+    Long createEntity(Comment entity, Long idNew);
```
Comment (manerajona) [inline src/main/java/com/alkemy/ong/core/usecase/CommentService.java]: llamar a idNew como newId para mantener constancia por favor

Comment (Juanchini23) [reply src/main/java/com/alkemy/ong/core/usecase/CommentService.java]: Cambios hechos!

**Comment by manerajona** [comment]
@Juanchini23 corregir conflictos porfa

**Pull request merged**